rest/adapter.go

```diff
@@ -69,6 +69,22 @@ func (p Prerequisite) convert() *evaluation.Prerequisite {
 	}
 }
 
+func (v VariationMap) convert() *evaluation.VariationMap {
+	return &evaluation.VariationMap{
+		TargetSegments: *v.TargetSegments,
+		Targets:        convertTargetToIdentifier(*v.Targets),
+		Variation:      v.Variation,
+	}
+}
+
+func convertTargetToIdentifier(tm []TargetMap) []string {
+	result := make([]string, 0)
```

Comment:
You could create a slice the same side as tm, to avoid allocations i.e. `result := make([]string, len(tm))`. Then in your range use the index to update the element in results.